### PR TITLE
Checkin

### PR DESCRIPTION
**Pull request by Aditiexcellence**

### CODE REVIEW

File: src/components/ExtraWorkFeedback/ExtraWorkFeedback.vue [hunk 1]
```diff
@@ -1,56 +1,62 @@
...
-      v-b-toggle="'collapse3'"
-      class="m-1 collapse-button"
-      v-bind:style="{'color': variant }"
-    >ERA</b-btn>
-    <b-collapse id="collapse3">
-      <b-container v-for="(era, index) in data.era" :key="index">
-        <div class="mt-2">
-          <span class="bold-text">{{ era.heading }}</span>
-          <h6>{{ era.content }}</h6>
+    <span v-if="data">
```
Comment (vueexcel) [inline src/components/ExtraWorkFeedback/ExtraWorkFeedback.vue]: *data* change it to something specific

File: src/components/Star/Star.vue [hunk 2]
```diff
@@ -1,39 +1,40 @@
...
-                v-bind:class="{ 'fa' : starKey <=  ratedStar}"
-                :style="{'font-size' : `${starSize}`}"
-                />
-            </h2>
-        </div>
+  <div class="d-flex justify-content-between mb-lg pb-2">
+    <div class="text-dark">
+      <h2>
+        <i
+          :disable="forDaisable === true ? true : false"
```
Comment (vueexcel) [inline src/components/Star/Star.vue]: *forDaisable* there is no need to use ternary operator, remove it, it doesn't make sense here

File: src/components/Star/Star.vue [hunk 3]
```diff
@@ -1,39 +1,40 @@
...
-    methods: {
-        rating(value) {
-            this.$emit('starRatingSelected',value)
-        }
+  name: "starRating",
+  props: {
+    displayStar: { type: Number, default: 5 },
+    ratedStar: { type: Number, default: 1 },
+    starSize: { type: String },
+    forDaisable: { type: Boolean }
```
Comment (vueexcel) [inline src/components/Star/Star.vue]: change this variable name *forDaisable*

File: src/components/weeklyReviewComponent/WeeklyReview/WeeklyReviewBox.vue [hunk 4]
```diff
@@ -1,54 +1,68 @@
...
+              <b-alert
+                show
+                class="Alert-class"
+              >Info: Overall rating/ comments mandatory for managers based on weekly report</b-alert>
+              <div class="mb-3">
+                <h6 class="rating-header mb-2">Overall Rating</h6>
+                <div class="border-top"></div>
+                <div class="mt-2 font-weight">Based on Weekly Review</div>
+              </div>
+              <span :disabled="reviews.reviewed == true ? true : false">
```
Comment (vueexcel) [inline src/components/weeklyReviewComponent/WeeklyReview/WeeklyReviewBox.vue]: again remove this ternary operator as well

File: src/components/weeklyReviewComponent/WeeklyReview/WeeklyReviewBox.vue [hunk 5]
```diff
@@ -1,54 +1,68 @@
...
+                  :ratedStar="ratedStarWeekly"
+                  :starSize="starSize"
+                  :forDaisable="reviews.reviewed"
+                  @starRatingSelected="submitStarRateWeekly"
+                />
+              </span>
+              <div
+                class="mt-2 font-weight"
+              >Difficulty level of Project (if project work you did was difficult/required more effort)</div>
+              <span :disabled="reviews.reviewed == true ? true : false">
```
Comment (vueexcel) [inline src/components/weeklyReviewComponent/WeeklyReview/WeeklyReviewBox.vue]: remove ternary operator again

File: src/components/weeklyReviewComponent/WeeklyReview/WeeklyReviewBox.vue [hunk 6]
```diff
@@ -1,54 +1,68 @@
...
+                  :forDaisable="reviews.reviewed"
+                  @starRatingSelected="submitStarRateDifficulty"
+                />
+              </span>
+              <div sm="6">
+                <h6 class="text-inverse">Comments</h6>
+              </div>
+              <b-form>
+                <b-form-textarea
+                  :disabled="reviews.reviewed == true ? true : false"
```
Comment (vueexcel) [inline src/components/weeklyReviewComponent/WeeklyReview/WeeklyReviewBox.vue]: ternary?

File: src/components/weeklyReviewComponent/WeeklyReview/WeeklyReviewBox.vue [hunk 7]
```diff
@@ -1,54 +1,68 @@
...
+                  :rows="3"
+                  v-model="text"
+                  id="default-textarea"
+                  placeholder="Performance or general comments (if any)..."
+                />
+                <b-button
+                  :disabled="reviews.reviewed == true ? true : false"
```
Comment (vueexcel) [inline src/components/weeklyReviewComponent/WeeklyReview/WeeklyReviewBox.vue]: ternary?

File: src/pages/Checkin/Checkin.vue [hunk 8]
```diff
@@ -44,18 +44,20 @@
...
-                      v-if="canShowreason"
-                      :class="{not_completed : report.task_completed === false}"
-                    >{{report.task_not_completed_reason}}</p>
-                    <p
-                      class="text-warning"
-                      v-if="canShowreason"
-                      :class="{not_completed : report.task_completed === false}"
-                    >{{report.highlight_task_reason}}</p>
-                    <p @click="showData" v-if="canShowless" class="text-primary btn">Read less..</p>
+                    <span v-if="report.task_not_completed_reason !== '' || report.highlight_task_reason ==!'' ">
```
Comment (vueexcel) [inline src/pages/Checkin/Checkin.vue]: *report.highlight_task_reason ==!''* ? this is supposed to be *report.highlight_task_reason !==''*

File: src/components/ExtraWorkFeedback/ExtraWorkFeedback.vue (hunk 1 above)
Comment (Aditiexcellence) [reply src/components/ExtraWorkFeedback/ExtraWorkFeedback.vue]: name change to user

File: src/components/Star/Star.vue (hunk 2 above)
Comment (Aditiexcellence) [reply src/components/Star/Star.vue]: removed ternary operator

File: src/components/Star/Star.vue (hunk 3 above)
Comment (Aditiexcellence) [reply src/components/Star/Star.vue]: changed variable name forDaisable to DisableStar

File: src/components/weeklyReviewComponent/WeeklyReview/WeeklyReviewBox.vue (hunk 4 above)
Comment (Aditiexcellence) [reply src/components/weeklyReviewComponent/WeeklyReview/WeeklyReviewBox.vue]: removed ternary operator

File: src/components/weeklyReviewComponent/WeeklyReview/WeeklyReviewBox.vue (hunk 5 above)
Comment (Aditiexcellence) [reply src/components/weeklyReviewComponent/WeeklyReview/WeeklyReviewBox.vue]: removed ternary operator

File: src/components/weeklyReviewComponent/WeeklyReview/WeeklyReviewBox.vue (hunk 6 above)
Comment (Aditiexcellence) [reply src/components/weeklyReviewComponent/WeeklyReview/WeeklyReviewBox.vue]: removed ternary operator

File: src/components/weeklyReviewComponent/WeeklyReview/WeeklyReviewBox.vue (hunk 7 above)
Comment (Aditiexcellence) [reply src/components/weeklyReviewComponent/WeeklyReview/WeeklyReviewBox.vue]: removed ternary operator

File: src/pages/Checkin/Checkin.vue (hunk 8 above)
Comment (Aditiexcellence) [reply src/pages/Checkin/Checkin.vue]: report.highlight_task_reason ==!'' changed to report.highlight_task_reason !==''

**Pull request merged**